docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst

```diff
@@ -0,0 +1,86 @@
+How to setup Single Sign-On (SSO) with GitHub, GitLab, or Bitbucket
+===================================================================
+
+.. include:: /shared/admonition-rtd-business.rst
+
+This how-to guide will provide instructions on how to enable :abbr:`SSO (Single Sign-on)` with GitHub, GitLab, or Bitbucket.
+If you want more information on this feature,
+please read :doc:`/commercial/single-sign-on`
+
+Prerequisites
+-------------
+
+Organization permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To change your Organization's settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`Username dropdown` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
+* Look at the **Owners** UI elements on the right menu.
+
+If you'd like to to modify this setting and are not an owner,
+you can ask an existing organization owner to follow this guide.
+
+User setup
+~~~~~~~~~~
+
+Users in your organization must have their GitHub, Bitbucket, or GitLab
+:doc:`account connected </connected-accounts>`,
+otherwise they will lose access to all repositories.
```

Comment:
What part don't you understand? If they don't have their account connected, they won't have access to anything on RTD, because that's how auth is handled.

Reply:
```suggestion
otherwise they won't have access to any project on Read the Docs.
```

I think that sounds less scary 😄

Reply:
thanks @humitos - now I think it reads without the scary ambiguity :)